docs/code-samples/getting-started/daml/Scripts/FixedRateBond.daml

```diff
@@ -0,0 +1,259 @@
+module Scripts.FixedRateBond where
+
+import DA.Date (DayOfWeek(..), Month(..), date, toGregorian)
+import DA.Map qualified as M (empty, fromList)
+import DA.Set qualified as S (empty, fromList, singleton, toList)
+import Daml.Script
+
+-- INTERFACE DEPENDENCIES --
+import Daml.Finance.Interface.Instrument.Bond.FixedRate.Factory qualified as FixedRate (Create(..), F(..))
+import Daml.Finance.Interface.Instrument.Bond.FixedRate.Types (FixedRate(..))
+import Daml.Finance.Interface.Lifecycle.Event qualified as Event (I)
+import Daml.Finance.Interface.Lifecycle.Rule.Claim qualified as Claim (ClaimEffect(..), I)
+import Daml.Finance.Interface.Lifecycle.Rule.Lifecycle qualified as Lifecycle (Evolve(..), I)
+import Daml.Finance.Interface.Settlement.Batch qualified as Batch (Settle(..))
+import Daml.Finance.Interface.Settlement.Instruction qualified as Instruction (Allocate(..), Approve(..))
+import Daml.Finance.Interface.Settlement.Types (Allocation(..), Approval(..))
+import Daml.Finance.Interface.Types.Common.Types (Id(..), InstrumentKey(..), Parties)
+import Daml.Finance.Interface.Types.Date.Calendar (BusinessDayAdjustment(..), BusinessDayConventionEnum(..), HolidayCalendarData(..))
+import Daml.Finance.Interface.Types.Date.Classes (HasUTCTimeConversion(..))
+import Daml.Finance.Interface.Types.Date.DayCount (DayCountConventionEnum(..))
+import Daml.Finance.Interface.Types.Date.RollConvention (Period(..), PeriodEnum(..), RollConventionEnum(..))
+import Daml.Finance.Interface.Types.Date.Schedule (Frequency(..), PeriodicSchedule(..), ScheduleFrequency(..))
+
+-- IMPLEMENTATION DEPENDENCIES --
+import Daml.Finance.Claims.Lifecycle.Rule (Rule(..))
+import Daml.Finance.Data.Reference.HolidayCalendar (HolidayCalendar(..))
+import Daml.Finance.Data.Time.DateClock.Types (Unit(..))
+import Daml.Finance.Data.Time.DateClockUpdate qualified as DataTime (DateClockUpdateEvent(..))
+import Daml.Finance.Instrument.Bond.FixedRate.Factory qualified as FixedRate (Factory(..))
+import Daml.Finance.Lifecycle.Rule.Claim qualified as Claim (Rule(..))
+
+import Scripts.Settlement (SettlementState(..), runSettlement)
+import Workflow.CreditAccount qualified as CreditAccount
+
+-- | Test script that
+-- 1. creates a fixed rate bond instrument and a holding on it
+-- 2. creates a lifecycle rule
+-- 3. creates a lifecycle event (time event: DateClockUpdate)
+-- 4. lifecycles the time event
+-- 5. processes the lifecycle effect
+-- 6. settles the distribution
+runFixedRateBond : Script()
+runFixedRateBond = do
+
+  -- Execute the `runSettlement` script to set up roles, accounts and a cash instrument
+  SettlementState{alice
```

Comment:
You are not using `alice`, `aliceAccount`, `aliceHolding` so no need to pattern match against those

Reply:
After moving to a separate project, this is now resolved, since we are currently not reusing anything.